README.md

```diff
@@ -8,6 +8,20 @@
 
 Operator to orchestrate [Move2Kube UI](https://github.com/konveyor/move2kube-ui) and [API](https://github.com/konveyor/move2kube-api).
 
+## Usage
+
+### Helm chart
+
+```
+helm repo add move2kube https://konveyor.io/move2kube
+helm repo update
+helm install --set ingresshost='my.k8s.cluster.domain.com' myrelease move2kube/move2kube
```

Comment:
Can we specify move2kube or move2kube release as the release name?

Reply:
done